packages/functions/src/notifications/AppointmentNotificationData.ts

```diff
@@ -44,3 +67,24 @@ function getUnsubscribeLink(donorId: string) {
     );
   }
 }
+
+function getAppointmentApprovalLink(
+  isProduction: boolean,
+  donorId: string,
+  appointmentId: string,
+  noShow: boolean
+) {
+  const parameters = `donorId=${donorId}&appointmentId=${appointmentId}&isNoshow=${noShow}`;
+
+  if (isProduction) {
+    return (
+      "https://us-central1-blood-components-9ad48.cloudfunctions.net/completeAppointmentApiHandler?" +
+      parameters
+    );
+  } else {
+    return (
+      "https://us-central1-blood-components.cloudfunctions.net/completeAppointmentApiHandler?" +
+      parameters
```

Comment:
Maybe it will be best to put the hosts in constants of PROD_HOST_URL and DEV_HOST_URL.

Reply:
Done